data/remnant/remnant missions.txt

```diff
@@ -730,7 +730,7 @@ event "remnant: nenia restored"
 mission "Remnant: Void Sprites 3"
 	landing
 	name "Visit void sprite planets"
-	description `Purchase a "gascraft" ship from the Remnant world of Viminal, and use it to explore the two gas giants where the void sprites live. Then, return to <planet>.`
+	description `Purchase a "puffin" ship from the Remnant world of Viminal, and use it to explore the two gas giants where the void sprites live. Then, return to <planet>.`
```

Comment:
Since Puffin is now a proper name instead of a description (as "gascraft" was), it should be capitalized.